libs/types/src/permissions.rs

```diff
@@ -399,14 +401,12 @@ where
 	}
 
 	pub fn contains(&self, tranche: TrancheId) -> bool {
-		self.info
-			.iter()
-			.position(|info| {
-				info.tranche_id == tranche && info.permissioned_till >= Now::now().as_secs().into()
-			})
-			.is_some()
+		self.info.iter().any(|info| {
+			info.tranche_id == tranche && info.permissioned_till >= Now::now().as_secs().into()
+		})
 	}
 
+	#[allow(clippy::result_unit_err)]
```

Comment:
I have been thinking about a better error resolution in the permissions pallet. Think this allow is a good hint to actually tackle that ^^